packages/itwin/grouping-mapping-widget/src/components/Properties/GroupProperties/GroupPropertyAction.tsx

```diff
@@ -260,25 +229,6 @@ export const GroupPropertyAction = ({
     }
   };
 
-  const startSearch = useCallback(() => {
-    if (!searchInput) return;
-    setActiveSearchInput(searchInput);
-    setSearched(true);
-  }, [searchInput]);
-
-  const clearSearch = useCallback(() => {
-    setSearchInput("");
-    setActiveSearchInput("");
-    setSearched(false);
-  }, []);
-
-  useEffect(() => {
-    if (searchInput.length === 0) {
-      setSearched(false);
-      clearSearch();
-    }
-  }, [searchInput, setSearched, clearSearch]);
-
   return (
     <DndContext
```

Comment:
I think DndContext should be scoped to the `GroupsPropertiesSelectionModal`, but that could be done in a separate PR.

Reply:
Sure, I will have to look into this further. There was a reason why I wrapped the parent of the modal but I can't remember why. It may not be needed anymore.

Reply:
Have you looked into this?

Reply:
Sorry, I meant to reply but I missed it. I was debugging an issue thinking it was related to the move of scope. But it's not. I believe I found a small bug relating to the presentation properties generation. I'm seeing a duplicate property. I will fix this in another PR. I have moved the context to the Modal scope.